### DISCUSSION TITLE
Scope management

### DISCUSSION THREAD

**Discussion by isitreallymyusername** [microsoft/dicom-server#591] [General]
Hi,

I may be interested in this service. I read that for the moment only scope "/" can be used. When do you plan to enhance the service so that it is possible to define subsets of images that can be accessed by a specific role ? 

Thanks

**Comment by jovinson-ms** [comment]
Hi there, we don't currently have a timeline for finer-grained access control. I'd be happy to discuss your use case further to better understand what you'd like to see!
